client/ayon_resolve/api/plugin.py

```diff
@@ -148,7 +150,7 @@ def load(self, files):
         # create mediaItem in active project bin
         # create clip media
         media_pool_item = lib.create_media_pool_item(
-            files,
+            files[0],
```

Comment:
So - what does this fix? It seems like an odd change to be related to "the new publisher" since it's a change for the loader?

Reply:
You're right, this is totally not related to the "new publisher" thing. 
Testing stuff on my side, I realized "loading as a clip" feature was broken on my branch, and this was my quick fix to get it sorted.

Thanks to your comment I digged into it a bit more. And it comes from this:
https://github.com/ynput/ayon-resolve/commit/d3ba10caf1a3b3b3990bf00b39772824bad64a3a?diff=unified&w=0#diff-bae424ef949b94556f4ade49faccf3ad7d73f3c16135597bf20a081f9ebaa711R206
`lib.create_media_pool_item` has changed to accept a file and not a list anymore. So I've replicated the previous logic one step above so the overall load logic does not change.